.drone.yml

```diff
@@ -68,11 +68,11 @@ steps:
   depends_on:
   - base
 
-- name: unit-tests-race
+- name: unit-tests%!(EXTRA string=-race)
```

Comment:
is this new?

Reply:
I think this is a bug in kres, we don't use `drone.yml` here, so it doesn't matter